general-superstaq/general_superstaq/user_config.py

```diff
@@ -153,7 +155,8 @@ def _config_to_yaml_str(config: Any) -> str:
         return self._client.aqt_upload_configs({"pulses": pulses_yaml, "variables": variables_yaml})
 
     def aqt_get_configs(self) -> Dict[str, str]:
-        """Calls private client to get the configurations for AQT
+        """Calls private client to get the configurations for AQT.
```

Comment:
nit: let's make this docstring more user-facing, e.g.
```suggestion
        """Retrieves the raw AQT config files that had previously been uploaded to Superstaq.
```